src/test/java/no/nav/tag/sykefravarsstatistikk/api/ApiTest.java

```diff
@@ -47,16 +47,19 @@ public void sammenligning__skal_returnere_riktig_objekt() throws Exception {
                 "  \"årstall\": 2019," +
                 "  \"virksomhet\": {" +
                 "    \"label\": \"NAV ARBEID OG YTELSER AVD OSLO\"," +
-                "    \"prosent\": 12.8" +
+                "    \"prosent\": 12.8," +
+                "    \"erMaskert\": false" +
                 "  }," +
                 "  \"næring\": {" +
                 "    \"label\": \"Trygdeordninger underlagt offentlig forvaltning\"," +
-                "    \"prosent\": 5.6" +
+                "    \"prosent\": 5.6," +
+                "    \"erMaskert\": false" +
                 "  }," +
                 "  \"sektor\": null," +
                 "  \"land\": {" +
                 "    \"label\": \"Norge\"," +
-                "    \"prosent\": 5.5" +
+                "    \"prosent\": 5.5," +
+                "    \"erMaskert\": false" +
                 "  }" +
                 "}"
         );
```

Comment:
Kan vi legge til en _end-to-end_ test av den slags som viser resultat ved maskerte SF prosent?

Reply:
Vil ikke det da det krever en del oppsett i mockingen. Vi lar ApiTest være en smoketest som bare tester det helt enkleste